engine/access/state_stream/account_status_filter.go

```diff
@@ -38,12 +38,37 @@ const (
 	CoreEventInboxValueClaimed = "flow.InboxValueClaimed"
 )
 
-var defaultCoreEventsMap map[string]struct{}
+var defaultCoreEventsMap map[string]map[string]struct{}
 
 func init() {
-	defaultCoreEventsMap = make(map[string]struct{}, len(DefaultCoreEvents))
-	for i := range DefaultCoreEvents {
-		defaultCoreEventsMap[DefaultCoreEvents[i]] = struct{}{}
+	defaultCoreEventsMap = make(map[string]map[string]struct{}, len(DefaultCoreEvents))
+
+	addFilter := func(eventType, field string) {
+		if _, ok := defaultCoreEventsMap[eventType]; !ok {
+			defaultCoreEventsMap[eventType] = make(map[string]struct{})
+		}
+		defaultCoreEventsMap[eventType][field] = struct{}{}
+	}
+
+	for _, eventType := range DefaultCoreEvents {
+
+		switch eventType {
+		case CoreEventAccountCreated,
+			CoreEventAccountKeyAdded,
+			CoreEventAccountKeyRemoved,
+			CoreEventAccountContractAdded,
+			CoreEventAccountContractUpdated,
+			CoreEventAccountContractRemoved:
+			addFilter(eventType, "address")
```

Comment:
since this is only run once, you can simplify this
```golang
switch eventType {
	case CoreEventAccountCreated:
		defaultCoreEventsMap[eventType] = map[string]struct{}{
			"address": {},
		}
}
```